README.md

```diff
@@ -171,7 +171,7 @@ On Linux
 export CMAKE_PREFIX_PATH="$(dirname $(which conda))/../" # [anaconda root directory]
 
 # Install basic dependencies
-conda install numpy pyyaml mkl setuptools cmake cffi
+conda install numpy pyyaml mkl setuptools cmake cffi typing
```

Comment:
Is that not a standard Python module? Can we add those annotations without adding more dependencies?

Reply:
Can we at least put the import in a try catch block and print a nice error message that explicitly shows the `pip` command you can use to install the module? Something like we do to remind people to update git submodules when they're missing

Reply:
Sounds good!